### PR TITLE
Allow client to run in isolation [WIP]

### PR DESCRIPTION
**Pull request by Reltre**
This PR adds to the Leaflet component so that it will use some dummy data when a server connection isn't present. I've included the script I used to generate that data, as well as the data file itself. The data file is a smaller version of the trees CSV file we have. I think it would also be fine to leave it out of this PR, and instruct anyone who wants to run the client without the server to use the script to generate the necessary dummy data.

### CODE REVIEW

**Comment by emillipede** [comment]
Thanks @Reltre - do you recommend that this view include a banner/footer that recognizes the user can't connect to the server?

**Comment by Reltre** [comment]
@emillipede Interesting idea. I'm not sure if we need a banner/footer, though I'm open to ideas if anyone thinks otherwise. This should only be an issue when working in development. We'll want to always ensure that the server and client sides are connected in production, and if they aren't then we will want to navigate a user to another page or show an empty map with no statistics.

**Comment by Reltre** [comment]
Looking over my implementation though and the current state of the Leaflet component, hmm, I would maybe like to see certain actions only taking place depending on the current development environment. Maybe we can set an environment variable to specify whether we are in TEST or not. If we are, then load up the dummy data, otherwise, we can connect to the server.

**Comment by ian-r-rose** [comment]
I think some kind of environment variable makes sense. One question: we plan to change the shape of the data significantly once we introduce things like filtering and joins. How do you envision this client-side only test data to respond to that?

File: src/Leaflet.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import { Map, TileLayer, Marker } from 'react-leaflet';
 import './Leaflet.css';
+import treeData from './data.json';
```
Comment (ian-r-rose) [inline src/Leaflet.js]: What does this import do to the size of the bundle? We may want to conditionally load it.

File: src/Leaflet.js [hunk 2]
```diff
@@ -82,6 +89,7 @@ export default class LeafletWrapper extends Component {
 
   render() {
     const markerList = this.state.markers.map(m => {
+      console.log(m);
```
Comment (ian-r-rose) [inline src/Leaflet.js]: Extra console.log

**Comment by Reltre** [comment]
I've amended the last commit to remove that logging expression. @ian-r-rose You're correct that this won't work if the shape of our data changes and when we want to use filters. I mainly want to use it as a starting point given the current data available.

As we discussed before, we should probably set up GraphQL for the frontend as well. This way, we won't have to generate various CSV files for different filtered data that needs testing on the frontend.

**Comment by Reltre** [comment]
@tpetersen0308 The current master build fixes the issues we were having, so be sure to pull that and you should be good to work on the frontend. I'm closing this PR in favor of the current master codebase.

**Comment by ian-r-rose** [comment]
Thanks @Reltre !

**Pull request closed without merging**